common/test/common/JsonComponentTest.scala

```diff
@@ -55,7 +57,7 @@ class JsonComponentTest extends FlatSpec with Matchers with ExecutionContexts {
 
     val result = Future {
       val request = FakeRequest("GET", "http://foo.bar.com/data.json")
-      JsonComponent(obj("name" -> "foo"))(request).result
+      JsonComponent(obj("name" -> "foo"))(request, play.api.libs.json.Writes.JsValueWrites).result
```

Comment:
I either need to provide the `Writes` explicitly or make the `request` implicit and not pass it explicitly. I'll opt for the latter as it is more representative of how it is used and have added the `withRefreshStatus`.